.pylintrc

```diff
@@ -298,7 +298,7 @@ max-module-lines=99999
 # spaces.  Google's externaly-published style guide says 4, consistent with
 # PEP 8.  Here, we use 2 spaces, for conformity with many open-sourced Google
 # projects (like TensorFlow).
```

Comment:
The last sentence mentions 2 spaces, which is no longer true. So maybe remove it?